tests/postpredict/dependence/test_build_train_X_Y.py

```diff
@@ -138,3 +138,98 @@ def test_build_train_X_Y_mask(obs_data, monkeypatch):
     
     assert_frame_equal(tdp.train_X, expected_train_X)
     assert_frame_equal(tdp.train_Y, expected_train_Y)
+
+
+def test_build_train_X_Y_pit_templates(obs_data, wide_model_out, monkeypatch):
+    # we use monkeypatch to remove abstract methods from the
```

Comment:
This is a neat trick! At some point, it might be worth creating a fixture for this pattern, since it's in the code base quite a few times.

Reply:
good idea, i filed issue #17 for that